packages/firefuel/test/src/firefuel_collection_test.dart

```diff
@@ -308,6 +346,44 @@ void main() {
     });
   });
 
+  group('#orderBy', () {
+    const testUsername1 = 'Alfred',
+        testUsername2 = 'Batman',
+        testUsername3 = 'Catwoman';
+    final testUser1 = TestUser(testUsername1),
+        testUser2 = TestUser(testUsername2),
+        testUser3 = TestUser(testUsername3);
+
+    setUp(() async {
+      await testCollection.create(testUser1);
+      await testCollection.create(testUser2);
+      await testCollection.create(testUser3);
+    });
+
+    test('should throw when no orderBys are given', () async {
+      expect(
+        () async => await testCollection.orderBy([]),
+        throwsA(isA<MissingValueException>()),
+      );
+    });
+
+    test('should return list users in asc order by name', () async {
```

Comment:
For the test name, try to not describe what you're doing in the test but what you actually expect for the thing you're testing to do 

```suggestion
    test('should return results in ascending order', () async {
```